sources/packages/backend/apps/api/src/route-controllers/overaward/models/overaward.dto.ts

```diff
@@ -30,7 +30,7 @@ export class StudentsOverawardAPIOutDTO {
   assessmentTriggerType?: AssessmentTriggerType;
 }
 
-export class AESTOverawardAPIOutDTO extends StudentsOverawardAPIOutDTO {
```

Comment:
It can remain as AESTOverawardAPIOutDTO as per comment https://github.com/bcgov/SIMS/pull/1906/files#r1178278574

As now AEST is the only client type who require AddedBy